huaweicloud/lb_v2_shared.go

```diff
@@ -280,7 +281,7 @@ func waitForLBV2viaPool(networkingClient *golangsdk.ServiceClient, id string, ta
 	if pool.Loadbalancers != nil {
 		// each pool has an LB in Octavia lbaasv2 API
 		lbID := pool.Loadbalancers[0].ID
-		return waitForLBV2LoadBalancer(networkingClient, lbID, target, nil, timeout)
+		return waitForLBV2LoadBalancer_v2(networkingClient, lbID, target, nil, timeout)
```

Comment:
`waitForLBV2LoadBalancer_v2` is not a good function name, it's up to you to rename it or not